src/com/egs/suzy/task2/UserDataHandler.java

```diff
@@ -0,0 +1,154 @@
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.lang.Integer.*;
+
+public class UserDataHandler {
+    private static final String[] COMMANDS = {"Add", "Remove", "List", "Exit"};
+    private static final int MAX_INCORRECT_INPUT = 3;
+    private static ArrayList<User> users = new ArrayList<>();;
+
+    UserDataHandler() throws IOException {
+       this(new File("users.txt"));
+    }
+
+    UserDataHandler(File userStorage) throws IOException {
+      /*  Runtime rs = Runtime.getRuntime();
```

Comment:
it's not good practice to start program execution from constructor. commandlistener could be public and be called